sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/text/TextTable.scala

```diff
@@ -39,10 +39,17 @@ case class TextTable(
   override def inferSchema(files: Seq[FileStatus]): Option[StructType] =
     Some(StructType(Array(StructField("value", StringType))))
 
-  override def newWriteBuilder(info: LogicalWriteInfo): WriteBuilder =
+  override def newWriteBuilder(info: LogicalWriteInfo): WriteBuilder = {
+    val writeInfo = LogicalWriteInfoImpl(
```

Comment:
shall we have one method in the parent class to create `LogicalWriteInfoImpl`?

Reply:
updated